stm32/Src/main.c

```diff
@@ -121,6 +127,8 @@ int main(void)
   //SensorsAdd(SDI12_Teros12Measure);
   //SensorsAdd(Phytos31_measure);
 
+  // Store the initial buffer state in FRAM
+  store_buffer_state(read_addr, write_addr, buffer_len);

```

Comment:
Why is the buffer state stored at this moment? Did we not just load it in the initialization?

Reply:
I'll remove this